test/fn/compile-app-settings.spec.js

```diff
@@ -158,6 +158,6 @@ describe('compile-app-settings', () => {
           .excludingEvery(['rules', 'contact_summary'])
           .to.deep.eq(expected);
       }
-    });
+    }).timeout(4000);
```

Comment:
I was re-running the tests many times locally and found a couple that routinely were timing out.  Seemed reasonable to bump up the timeout on these tests so they were not so flaky.....